src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryCustomObject.cs

```diff
@@ -15,13 +15,39 @@ internal static partial class Library
     {
         internal class JsonCustomObject : ICustomObject
         {
+            private static readonly FormulaType _objectType = new ExternalType(ExternalTypeKind.Object);
+            private static readonly FormulaType _arrayType = new ExternalType(ExternalTypeKind.Array);
```

Comment:
Should this be static fields on the ExternalType class itself? They're generally useful.

Reply:
Done